teps/0128-scheduled-runs.md

```diff
@@ -199,8 +200,73 @@ to address specifically.
 feature like [polling runs](./0083-polling-runs-in-tekton.md). This proposal doesn't prevent that,
 and ScheduledTriggers could be leveraged to implement similar functionality in a higher level system.
 
+- This seems to templatisation over templatisation. Pipelinerun already give us flexibility to pass parameters for Pipeline based on requirements. TriggerBindings and TriggerTemplates were designed for templatising Pipelineruns based on events coming. In this case, they will always remain same.
+
 ## Alternatives
 
+### Add a field or annotation in Pipelinerun for scheduling
+
+In this case we add a controller in experimental repo which can later be added to pipelines. Or it can be part of Pipelinerun controller.
+We will add field for scheduling:
+```yaml
+apiVersion: tekton.dev/v1
+kind: Pipelinerun
+metadata:
+  generateName: nightly-release
+spec:
+  schedule: "0 0 * * *"
+  pipelineRef:
+    name: "release"
+  taskRunTemplate:
+    serviceAccountName: ...
+```
+Or an annotation:
+```yaml
+apiVersion: tekton.dev/v1
+kind: Pipelinerun
+metadata:
+  generateName: nightly-release
+  annotations:
+    schedule: "0 0 * * *"
+spec:
+  pipelineRef:
+    name: "release"
+  taskRunTemplate:
+    serviceAccountName: ...
+```
+
+
+The controller will watch Pipelinerun and generate jobs based on annotation or field.
```

Comment:
In this solution, would the pipelinerun controller have to understand to skip pipelineruns with the "schedule" annotation or field, since the new controller would be responsible for creating more pipelineruns?

Reply:
Yes, I need to make this more explicit.

Reply:
would you mind adding more detail on this to the proposal?

Reply:
Added that detail